lmdeploy/lite/apis/auto_awq.py

```diff
@@ -26,24 +28,42 @@
 
 
 def auto_awq(model: str,
+             work_dir: str,
```

Comment:
What's the motivation of changing `work_dir` from optional argument to positional argument? I tried the following codes, it  raised an error:

```plaintext
(triton) [lvhan@SH-IDC1-10-140-24-140 llmdeploy]$ python lmdeploy/lite/apis/auto_awq.py --model /nvme/shared_data/llama2/huggingface/llama-2-7b-chat/
[2023-08-21 11:47:05,460] [INFO] [real_accelerator.py:133:get_accelerator] Setting ds_accelerator to cuda (auto detect)
ERROR: The function received no value for the required argument: work_dir
Usage: auto_awq.py MODEL WORK_DIR <flags>
  optional flags:        --w_bits | --w_sym | --w_group_size | --device

For detailed information on this command, run:
  auto_awq.py --help
```